docs/source/contribution/development_for_databricks.md

```diff
@@ -0,0 +1,73 @@
+# Developing Kedro for Databricks
+
+[Databricks](https://www.databricks.com/) is a cloud-based platform for data engineering and data science that many Kedro users deploy their Kedro projects to. Extending and improving the Kedro experience on Databricks is important to a large part of Kedro's user base.
+
+## Introduction
+
+This page describes how to efficiently develop features and fixes for Kedro on Databricks. Using this guide, you will be able to quickly test your locally modified version of Kedro on Databricks as part of a build-and-test development cycle.
+
+## Prerequisites
+
+You will need the following prerequisites to follow this guide:
+
+1. Python **version >=3.8** installed.
+2. A Python virtual environment activated in which the [Databricks CLI](https://docs.databricks.com/dev-tools/cli/index.html) is installed with [authentication for your workspace](https://docs.databricks.com/dev-tools/cli/index.html#set-up-the-cli).
+3. Access to a Databricks workspace with an [existing cluster](https://docs.databricks.com/clusters/create-cluster.html).
```

Comment:
```suggestion
3. Access to a Databricks workspace on an [existing cluster](https://docs.databricks.com/clusters/create-cluster.html).
```

Reply:
No mention of type of cluster, e.g. AWS? We have a deployment guide and it seems to only support AWS at present, so is that a prerequisite or can any cloud provider cluster be used? Probably worth saying or pointing to a relevant databricks link?

Reply:
This is a good point, though creating a cluster on Databricks is actually a cloud-agnostic process on Databricks. The linked documentation actually covers the topic really well and the user can switch the cloud provider in the top right to view the (very minor) differences for their specific cloud. For this reason I don't think it's important that we specify a cloud here.